src/main/java/com/users/specification/UserSpecification.java

```diff
@@ -53,6 +53,30 @@ public Predicate toPredicate(Root<User> root, CriteriaQuery<?> query, CriteriaBu
             }
         }
 
+        if (criteria.getSecondaryEmail() != null) {
```

Comment:
astea le ai incercat?

Reply:
nu, asta se pare ca nu e adaptat sa mi caute cum vreau, dar se face

Reply:
este ok, l am testat acum, daca dau null mi le aduce pe toate in functie de page si size, daca ii dau ceva cu contains sau equals imi aduce corect